### PR TITLE
:bug: fix upload

### PR DESCRIPTION
**Pull request by taoqf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -299,7 +127,7 @@ return RNFS.unlink(path)
 
 ```javascript
 // require the module
-var RNFS = require('react-native-fs');
+var RNFS = require('@taoqf/react-native-fs');
```
Comment (FDiskas) [inline README.md]: 👎

File: package.json [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
-    "react-native-windows": "^0.57.0"
-  },
-  "peerDependencies": {
-    "react": "^16.2.0",
-    "react-native": ">=0.51.0 <1.0.0",
-    "prop-types": "^15.6.0"
-  }
-}
+	"name": "react-native-fs",
+	"version": "1.0.3",
```
Comment (FDiskas) [inline package.json]: do not change version number

File: package.json [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
+		"prop-types": "15.6.0",
+		"create-react-class": "^15.6.2",
+		"react-native-windows": "^0.57.0"
+	},
+	"peerDependencies": {
+		"react": "^16.2.0",
+		"react-native": ">=0.51.0 <1.0.0",
+		"prop-types": "^15.6.0"
+	}
+}
```
Comment (FDiskas) [inline package.json]: revert spacing

File: README.md [hunk 4]
```diff
@@ -1,237 +1,64 @@
...
-```
-
 ## Usage (Windows)
 
 ### Adding automatically with react-native link
 
 The `link` command also works for adding the native dependency on Windows:
 
-`react-native link react-native-fs`
+`react-native link @taoqf/react-native-fs`
```
Comment (FDiskas) [inline README.md]: revert

File: README.md [hunk 5]
```diff
@@ -1,237 +1,64 @@
...
+Follow the instructions in the ['Linking Libraries'](https://github.com/Microsoft/react-native-windows/blob/master/docs/LinkingLibrariesWindows.md) documentation on the react-native-windows GitHub repo. For the first step of adding the project to the Visual Studio solution file, the path to the project should be `../node_modules/@taoqf/react-native-fs/windows/RNFS/RNFS.csproj`.
 
 ## Examples
 
 ### Basic
 
 ```javascript
 // require the module
-var RNFS = require('react-native-fs');
+var RNFS = require('@taoqf/react-native-fs');
```
Comment (FDiskas) [inline README.md]: revert

File: README.md [hunk 6]
```diff
@@ -262,7 +89,7 @@ RNFS.readDir(RNFS.MainBundlePath) // On Android, use "RNFS.DocumentDirectoryPath
...
```
Comment (FDiskas) [inline README.md]: revert

File: README.md [hunk 7]
```diff
@@ -1,237 +1,64 @@
...
 
-###  Adding with CocoaPods
+### Adding with CocoaPods
 
  Add the RNFS pod to your list of application pods in your Podfile, using the path from the Podfile to the installed module:~~
 
-```
-pod 'RNFS', :path => '../node_modules/react-native-fs'
+```pod
+pod 'RNFS', :path => '../node_modules/@taoqf/react-native-fs'
```
Comment (FDiskas) [inline README.md]: revert

File: README.md [hunk 8]
```diff
@@ -1,237 +1,64 @@
...
-
-As @a-koka pointed out, you should then update your package.json to
-`"react-native-fs": "2.0.1-rc.2"` (without the tilde)
 
 ### Adding automatically with react-native link
 
 At the command line, in your project folder, type:
 
-`react-native link react-native-fs`
+`react-native link @taoqf/react-native-fs`
```
Comment (FDiskas) [inline README.md]: revert

File: README.md [hunk 9]
```diff
@@ -1,237 +1,64 @@
...
+# react-native-fs
+
+Native filesystem access for react-native, fork from [react-native-fs](https://www.npmjs.com/package/react-native-fs)
 
 ## Usage (iOS)
 
 First you need to install react-native-fs:
 
+```sh
+npm install @taoqf/react-native-fs --save
```
Comment (FDiskas) [inline README.md]: revert

File: README.md (hunk 1 above)
Comment (taoqf) [reply README.md]: Sorry for that, It has been a long time, and I really do not know if `react-native-fs` would work now or not and I don't have time to do some tests.
And, I could not resolve the conflicts in `android/src/main/java/com/rnfs/RNFSManager.java`.

**Comment by taoqf** [comment]
I am sorry I don't has time on this, and I don't know how to resolve the conflicts.

**Pull request closed without merging**